website/templates/project/view_file.mako

```diff
@@ -182,7 +182,7 @@
             docId: ${ sharejs_uuid | sjson, n },
             userId: ${ user['id'] | sjson, n },
             userName: ${ user['fullname'] | sjson, n },
-            userUrl: ${ '/' + user['id'] + '/' | sjson, n },
+            userUrl: ${ '/' + (user['id'] or '') + '/' | sjson, n },
```

Comment:
That's a fair question. Given that we use 5-character user IDs, are there circumstances where user['id'] would be falsy for a logged in user?

When the user is not logged in, user['id`] = None. Before the markupsafe conversion, we used to construct URLs the [crazy way](https://github.com/CenterForOpenScience/osf.io/blob/master/website/templates/project/view_file.mako#L176); I'm not sure the URL field was ever meaningful in this case.

Here's what the URL looks like on production:
```
        editor: {
            registration: true,
            docId: 'None',
            userId: 'None',
            userName: '',
            userUrl: '/None/',
            userGravatar: 'https://secure.gravatar.com/avatar/6adf97f83acf6453d4a6a4b1070f3754?d=identicon&size=25'.replace('&amp;', '&')
        },
```